dist/debian/DEBIAN_PACKAGE.md

```diff
@@ -15,6 +15,15 @@ cp -r dist/debian .
 Next, make sure all the dependencies are met. See
 [INSTALL.md](../../INSTALL.md#install-system-dependencies)
 for the basic apt-get install list.
+
+To build with libsubid support add the following to the control file (requires
+at least Ubuntu Noble or Debian Bookworm):
+
+```
+Build-Depends:
+ libsubid-dev
+```
```

Comment:
This looks good to me except that lint markdown is complaining about a lack of language specification on the code block.  The trouble is it's not clear what to use.  The [markdown documentation](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) references [a languages YAML file](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml) but it's not clear what keywords to use.  I suggest trying `deb` although it might be `dsc`.

Reply:
Done. added dsc.